src/expr/core/src/expr/expr_udf.rs

```diff
@@ -144,8 +144,7 @@ impl UdfExpression {
             );
         }
 
-        let data_chunk =
-            DataChunk::try_from(&output).expect("failed to convert UDF output to DataChunk");
+        let data_chunk = DataChunk::try_from(&output)?;
```

Comment:
What happens if we only change this line without changing udf sdk? 🤔

Reply:
will result in expression error: `invalid decimal`, or silently produce wrong value `1e-10` -> `1.000...`